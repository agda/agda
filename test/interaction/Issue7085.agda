
postulate A : Set

module M where
  postulate a : A

x : A
x = {! M.a !} -- No solution found

open M

y : A
y = {! M.a !} -- works
