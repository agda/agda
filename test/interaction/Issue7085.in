top_command (cmd_load currentFile [])
goal_command 0 cmd_autoOne "M.a"
goal_command 1 cmd_autoOne "M.a"
