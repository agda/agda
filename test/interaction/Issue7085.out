(agda2-status-action "")
(agda2-info-action "*Type-checking*" "" nil)
(agda2-highlight-clear)
(agda2-status-action "")
(agda2-info-action "*All Goals*" "?0 : A ?1 : A " nil)
((last . 1) . (agda2-goals-action '(0 1)))
(agda2-give-action 0 "M.a")
((last . 1) . (agda2-goals-action '(1)))
(agda2-give-action 1 "a")
((last . 1) . (agda2-goals-action '()))
